manifests/server/kickstart/runpuppet.pp

```diff
@@ -0,0 +1,86 @@
+# This class manages the runpuppet script, which is a script that can be run
+# to bootstrap provisioned systems, adding them to puppet and running it in a
+# fashion similar so `simp bootstrap`.
+#
+# @param data_dir The location of the web root in which runpuppet wil be placed
```

Comment:
I don't think this parameter is needed.  It is only used to set another parameter, location.